src/screens/Voucher/VoucherHomeScreen.tsx

```diff
@@ -37,11 +50,40 @@ export const VoucherHomeScreen = ({ navigation, route }: StackScreenProps<any>)
   // refresh if the refreshAuth param changed, which happens after login
   useEffect(refreshAuth, [route.params?.refreshAuth]);
 
+  const { mutate: mutateLogIn } = useMutation(logIn);
+
+  useEffect(() => {
+    const accountCheck = async () => {
+      const loginData = await voucherMemberLoginInfo();
+      const savedDate = (await readFromStore(SAVED_DATE)) || moment().format('YYYY-MM-DD');
+
+      if (loginData && !moment().isSame(savedDate, 'day')) {
+        mutateLogIn(JSON.parse(loginData as string) as VoucherLogin, {
+          onSuccess: (responseData) => {
+            if (!responseData?.member) {
+              storeVoucherAuthToken();
+              storeVoucherMemberId();
+              storeVoucherMemberLoginInfo();
+              refresh();
+            }
+          }
+        });
+
+        addToStore(SAVED_DATE, moment().format('YYYY-MM-DD'));
+        setAccountCheckLoading(false);
+      } else {
+        setAccountCheckLoading(false);
+      }
+    };
+
+    accountCheck();
+  }, []);
+
   const refreshHome = useCallback(async () => {
     await refetchHomeText();
   }, []);
 
-  if (loadingHomeText) {
+  if (loadingHomeText || accountCheckLoading) {
```

Comment:
what about `loadingAccountCheck` to be aligned with `loadingHomeText`?

Reply:
It's definitely more accurate this way. 👍